src/components/MovieDetails/MovieDetailsBody/MovieDetailsBody.tsx

```diff
@@ -16,7 +16,9 @@ function MovieDetailsBody() {
         </div>
         <div className={styles.movieDetails_rightCol}>
           <div className={styles.movieDetails_bodyWrapper}>
-            <div className={styles.movieDetails_title}>{movie.title}</div>
+            <div id={"movieDetailsTitle"} className={styles.movieDetails_title}>
```

Comment:
There is a convention for id, if you need an id for cypress, please use this `data-cy='movieDetailsTitle'`